guide/additional-info/changes-in-v12.md

```diff
@@ -1819,11 +1819,11 @@ All the `.send***()` methods have been removed in favor of one general `.send()`
 
 #### User#addFriend
 
-`user.addFriend()` has been removed entirely, along with all other user account-only methods.
+`user.addFriend()` was removed entirely, along with all other user account-only methods.
 
 #### User#avatarURL
 
-`user.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If the `dynamic` option is provided you will receive a `.gif` URL if the image is animated, otherwise it will fall back to the specified `format` or its default `.webp` if none is provided.
+`user.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If the `dynamic` option is provided, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp` if none is provided.
```

Comment:
```suggestion
`user.avatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If you provide the `dynamic` option, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp`.
```